tests/html-nested-sort-percentage/reference/clang-10/coverage.A.7fc56270e7a70fa81a5935b72eacbe29.html

```diff
@@ -23,7 +23,7 @@ <h1>GCC Code Coverage Report</h1>
     </tr>
     <tr>
       <th scope="row">Date:</th>
-      <td>0000-00-00 00:00:00</td>
+      <td>00-00-00 00:00:00</td>
```

Comment:
is this change expected?

Reply:
I also saw this when creating the single file report and fixed the error in the scrubber there. See https://github.com/gcovr/gcovr/pull/916/files#diff-14fc10b334e44e45733e71ed3f0d748422b8f8dedc6078c92f2ebca731072ccc.